pymc3/tests/test_distributions.py

```diff
@@ -2051,7 +2051,6 @@ def test_wishart(self, n):
         pass
 
     @pytest.mark.parametrize("x,eta,n,lp", LKJ_CASES)
-    @pytest.mark.xfail(reason="Distribution not refactored yet")
     def test_lkj(self, x, eta, n, lp):
```

Comment:
@ricardoV94 is the case `(np.array([0.7,0,-0.7]), 0, 3, -np.inf)` here a extreme corner case ? 

It looks like previously, this tests didn't pass through `random` (which now is it's `rng_fn`) for `logp`, now it does. And hence fails.